core/app/models/spree/return_item.rb

```diff
@@ -168,8 +168,11 @@ def currency
 
     def process_inventory_unit!
       inventory_unit.return!
-
-      Spree::StockMovement.create!(stock_item_id: stock_item.id, quantity: 1) if should_restock?
+      Spree::StockMovement.create!({
+        stock_item_id: stock_item.id,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.